LiskHQ/lisk-desktop
# Ignore inner text case in e2e-testing - Closes #34

### What was the problem?
We used Chai for assertion and is checked the inner text of elements to be exactly equal to given value. and since we had LOGOUT and SEND in uppercase in previous design, e2e tests failed because of slight styling change.

### How did I fix it?
I use the method `equalIgnoreCase` from `chai-string` instead.

### How to test it?
Change the case of inner text of a random element whose text we're checking in e2e tests, and see if that specific test failed or not.

### Review checklist
- [x] The PR solves #34
